### PR TITLE
Swagger Update

### PR DESCRIPTION
**Pull request by akroon3r**
Client requires updated data structure for new models/schemas as well as updates to current models/schemas.

### CODE REVIEW

File: documentation/open-api/swagger.yaml [hunk 1]
```diff
@@ -17,6 +17,10 @@ tags:
   description: Everything about invigilators
 - name: Rooms
   description: Everything about rooms
+- name: Appointments
+  description: Everything about appointments
+- name: ExamTypes
```
Comment (sjrumsby) [inline documentation/open-api/swagger.yaml]: Should `ExamTypes` be part of exams?

Comment (akroon3r) [reply documentation/open-api/swagger.yaml]: Screen Shot 2019-03-29 at 9 31 51 AM

Comment (akroon3r) [reply documentation/open-api/swagger.yaml]: Did it based on the folder structure we have in resources.

Comment (akroon3r) [reply documentation/open-api/swagger.yaml]: Also, exam_types is its own model, just my reasoning behind it.

Comment (sjrumsby) [reply documentation/open-api/swagger.yaml]: Weird. I wonder why we did it that way. This makes sense based on that I guess

**Pull request merged**